### PR TITLE
chore(analytics): add ownerType value to home view viewAll signals

### PR DESCRIPTION
**Pull request by dblandin**
Add `ownerType` to viewAll behaviors with an explicit `href` provided so we can reliably pass this value through to analytics event payloads.

### CODE REVIEW

File: _schemaV2.graphql [hunk 1]
```diff
@@ -10912,7 +10912,7 @@ type HomeViewComponent {
   description: String
 
   # A screen to navigate to when this component is clicked
-  href: String
+  href: String @deprecated(reason: "Use `behaviors.viewAll.href` instead")
```
Comment (anandaroop) [inline _schemaV2.graphql:10915]: **thought (non-blocking):** Thanks! I think we could go one step further and remove this, once we've verified Eigen doesn't currently use it. Might as well clean up cruft while we're still [allowed to make breaking changes](https://www.notion.so/artsy/Art-Advisor-HomeView-Rollout-7b674fb1371c4d3e864fd08111f56644?pvs=4#48ca0c764af445cb955a04a8a8cc71fc) — but that can be a separate PR.

Comment (dblandin) [reply _schemaV2.graphql:10915]: Yeah, I'm pretty sure we don't reference it any more but I'll make sure and follow-up with the removal in a separate PR.
